commit 3663b980be8bb30bcaf69140d8b93944277f334c (refs/changes/02/2902/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-11-03T22:06:34+00:00 (1 year, 11 months ago)
    
    TOOLS-1677 cr.joyent.us needs to work around Gerrit issue 1451
    TOOLS-1915 Update Gerrit to link to jira.joyent.us

diff --git a/images/appserver/etc/gerrit.config.base b/images/appserver/etc/gerrit.config.base
index 79b5d2f..5482674 100644
--- a/images/appserver/etc/gerrit.config.base
+++ b/images/appserver/etc/gerrit.config.base
@@ -67,11 +67,11 @@
 # trackers.
 #
 [commentlink "devjira"]
-	match = ([A-Z0-9]+-[0-9]+)
-	link = https://devhub.joyent.com/jira/browse/$1
+	match = "(^|\\s|[:.,\\(\\[\\{])([A-Z]+-[0-9]+)\\b"
+	html = "$1<a href=\"https://jira.joyent.us/browse/$2\">$2</a>"
 [commentlink "github"]
-	match = "([a-zA-Z0-9]+)/([-_a-zA-Z0-9]+)#([0-9]+)"
-	link = https://github.com/$1/$2/issues/$3
+	match = "(^|\\s|[:.,\\(\\[\\{])([a-zA-Z0-9]+)/([-_a-zA-Z0-9]+)#([0-9]+)\\b"
+	html = "$1<a href=\"https://github.com/$2/$3/issues/$4\">$2/$3#$4</a>"
 
 #
 # Clarify the purpose of some of the buttons on the Change screen.
